tools/convert_tests/convert_tests_helper.py

```diff
@@ -0,0 +1,409 @@
+from datetime import datetime
+import pytz
+import re
+
+# Define short names for various data types
+short_name_map = {
+    "string": "str",
+    "boolean": "bool",
+    "varbinary": "vbin",
+    "timestamp": "ts",
+    "timestamp_tz": "tstz",
+    "interval_year": "iyear",
+    "interval_day": "iday",
+    "interval": "iday",
+    "decimal": "dec",
+    "precision_timestamp": "pts",
+    "precision_timestamp_tz": "ptstz",
+    "fixedchar": "fchar",
+    "varchar": "vchar",
+    "fixedbinary": "fbin",
+}
+
+long_name_map = {v: k for k, v in short_name_map.items()}
+
+# Mapping of timezone abbreviations to pytz time zones
+timezone_abbr_map = {
+    "PST": "America/Los_Angeles",
+    "EST": "America/New_York",
+    "CST": "America/Chicago",
+    "MST": "America/Denver",
+    "HST": "Pacific/Honolulu",
+    "UTC": "UTC",
+}
+
+SQUOTE = "\u200B"
+DQUOTE = "&"
+
+# Map timezone abbreviations to valid pytz timezone names
+timediff_abbr_map = {
+    "-0800": "US/Pacific",  # Pacific Standard Time
+    "-0500": "US/Eastern",  # Eastern Standard Time
+}
+
+timediff_zone_map = {
+    "-0800": "PST",  # Pacific Standard Time
+    "-0500": "EST",  # Eastern Standard Time
+}
+
+
+def convert_type(type_str, mapping):
+    """Helper function to convert a type string using the provided mapping."""
```

Comment:
Perhaps put the string type in the function signature so it's not needed here.  That way the phrase "type string" doesn't sound like the type "string".  It would be useful to put as many types as possible in function signatures.

Reply:
calling this value_type to avoid confusion.